platform/javascript/detect.py

```diff
@@ -48,11 +48,6 @@ def get_flags():
     return [
         ("tools", False),
         ("builtin_pcre2_with_jit", False),
-        # Disabling the mbedtls module reduces file size.
-        # The module has little use due to the limited networking functionality
-        # in this platform. For the available networking methods, the browser
-        # manages TLS.
-        ("module_mbedtls_enabled", False),
```

Comment:
Should this be changed in `3.x` too?

Reply:
I'm actually thinking about it.
This was originally done to save >1MiB for OpenSSL for no real gain (SSL is still handled by the browser).
Now that we use mbedTLS which is much smaller, and the Crypto module has many more functionalities, I feel like we should re-evaluate it.
I think I'll open a proposal for that.

Reply:
If it's not needed for this PR to build, I'd suggest leave this change out and move to a separate PR to keep the discussion focused.

FWIW I also think it would be fine to keep it enabled for JS templates, we just need to measure size impact but it's likely minimal.

Reply:
> If it's not needed for this PR to build, I'd suggest leave this change out and move to a separate PR to keep the discussion focused.

Well, it does build without it, but the editor will not run.
So I guess it's fine for now, and then make a separate PR.